sklearn/gaussian_process/_gpr.py

```diff
@@ -48,16 +48,17 @@ class GaussianProcessRegressor(MultiOutputMixin,
         passed, the kernel "1.0 * RBF(1.0)" is used as default. Note that
         the kernel's hyperparameters are optimized during fitting.
 
-    alpha : float or array-like of shape (n_samples), default=1e-10
+    alpha : float, default=1e-10
         Value added to the diagonal of the kernel matrix during fitting.
-        Larger values correspond to increased noise level in the observations.
-        This can also prevent a potential numerical issue during fitting, by
+        This can prevent a potential numerical issue during fitting, by
         ensuring that the calculated values form a positive definite matrix.
-        If an array is passed, it must have the same number of entries as the
-        data used for fitting and is used as datapoint-dependent noise level.
-        Note that this is equivalent to adding a WhiteKernel with c=alpha.
-        Allowing to specify the noise level directly as a parameter is mainly
-        for convenience and for consistency with Ridge.
+        It can also be interpreted as the variance of Gaussian measurement
+        noise on the training observations (although this is not the same as
+        using the WhiteKernel). If an array is passed, it must have the same
```

Comment:
```suggestion
        It can also be interpreted as the variance of additional Gaussian measurement
        noise on the training observations. Note that this is different from
        using a `WhiteKernel`. If an array is passed, it must have the same
```

Reply:
Yep, fine by me